src/Ledger/hs-src/test/UtxowSpec.hs

```diff
@@ -57,6 +58,7 @@ data SimpleTxBody = MkSimpleTxBody
 bodyFromSimple :: PParams -> SimpleTxBody -> TxBody
 bodyFromSimple pp stxb = let s = 5 in MkTxBody
   { txins  = stxins stxb
+  , refInputs = srefInputs stxb
```

Comment:
indentation

Reply:
Okay, I have aligned the `=` symbols in this and other definitions for consistency.  However, note that previous commits did not have perfectly consistent alignment, so this change means the PR now touches some lines that are not necessarily within the task scope.